staging/src/kubevirt.io/client-go/log/log.go

```diff
@@ -374,6 +374,13 @@ func LogLibvirtLogLine(logger *FilteredLogger, line string) {
 	pos := strings.TrimSpace(fragments[3])
 	msg := strings.TrimSpace(fragments[4])
 
+	//TODO: implement proper behavior for unsupported GA commands
+	// by either considering the GA version as unsupported (don't send any command)
+	// or just don't send commands which not supported
+	if strings.Contains(msg, "unable to execute QEMU agent command") {
+		severity = "info"
```

Comment:
Can you use `severity=LogLevelNames[INFO]` or `severity=LogLevelNames[WARNING]`?

Reply:
@rmohr Done, thanks!